### PR TITLE
Started to write a README

### PR DESCRIPTION
**Pull request by m0veax**
As requested in #3 I started writing a README File that documents how to start the application locally

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
+# buildjs
+pkg/
```
Comment (orangecms) [inline .gitignore]: there is also a gitignore in `parser/`, we already have this there

File: README.md [hunk 2]
```diff
@@ -1,34 +1,19 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+A graphical device tree visualizer (early development) 
 
-## Getting Started
+## Local Deployment
```
Comment (orangecms) [inline README.md]: change into _Development_ :)

File: README.md [hunk 3]
```diff
@@ -1,34 +1,19 @@
...
+
+Install wasm-pack
```
Comment (orangecms) [inline README.md]: let's put the `cargo install` command here, and add npm as well

something like

### Dependencies

Install `wasm-pack` and npm dependencies:

```sh
cargo install wasm-pack
npm install
```

### Run the app in development mode

```sh
npm start
```

**Comment by orangecms** [comment]
We started a description of what a device tree is already a bunch of times.

I had that on my slides in a bunch of talks as you may remember, and we just added a bit to the [device_tree-rs](https://github.com/platform-system-interface/device_tree-rs) crate. Let's add a reference to that here as well. :)

**Comment by orangecms** [comment]
see also https://github.com/platform-system-interface/device_tree-rs/issues/2

**Comment by m0veax** [comment]
I resolved your comments and added the references you thought about :) Let me know if we need change anything further

**Pull request merged**